core/templates/dev/head/header_css_libs.html

```diff
@@ -25,5 +6,13 @@
       href="https://fonts.googleapis.com/css?family=Roboto">
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
+{% if DEV_MODE %}
+<link rel="stylesheet" type="text/css" media="screen"
```

Comment:
indent this and next line by two, similar for else block

(In this codebase we use indentation to optimize for code readability than for the readability of the generated html, since the former happens more frequently.)